src/launcher.c

```diff
@@ -293,10 +314,29 @@ static void *load_app(char *name)
     }
   }
 
-  if (patch_svc(code, size) != 0) {
+
+
+  uint32_t start = 0;
+  uint32_t code_length = size;
+  if (app->elf.code_start && app->elf.code_end) {
```

Comment:
`app->elf.code_start != 0 && app->elf.code_end != 0`
I think that implicit boolean cast for integers is confusing, this is more explicit and clear

Reply:
i agree :bow: